src/components/InputPhone/helpers/getCountryFromValue.ts

```diff
@@ -1,5 +1,13 @@
+import * as RNLocalize from 'react-native-localize';
```

Comment:
Maybe you can use NativeModules instead of installing a third-party library that applies size overhead